src/test/java/io/vertx/core/spi/tracing/LocalEventBusTracerTest.java

```diff
@@ -32,15 +32,15 @@ public void testInboundInterceptor() throws Exception {
     tracer = new VertxTracer() {};
     vertx2.eventBus().addInboundInterceptor(deliveryCtx -> {
       ContextInternal ctx = (ContextInternal) vertx.getOrCreateContext();
-      ctx.localContextData().put("key", "val");
+//      ctx.putLocal(FakeTracer.ACTIVE_SCOPE_KEY, "val");
       deliveryCtx.next();
     });
     Context receiveCtx = vertx2.getOrCreateContext();
     CountDownLatch latch = new CountDownLatch(1);
     receiveCtx.runOnContext(v -> {
       vertx2.eventBus().consumer("the_address", msg -> {
-        Object val = ((ContextInternal) vertx.getOrCreateContext()).localContextData().get("key");
-        assertEquals("val", val);
+//        Object val = vertx.getOrCreateContext().getLocal(FakeTracer.ACTIVE_SCOPE_KEY);
```

Comment:
Same question

Reply:
test is actually ignored :-) so previous code will not work anymore when the deprecated code will be removed, I sketched what should be correct more or less